go.mod

```diff
@@ -16,9 +16,9 @@ require (
 	go.uber.org/atomic v1.10.0
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 	gorm.io/datatypes v1.0.7
-	gorm.io/driver/mysql v1.4.3
-	gorm.io/driver/postgres v1.3.10
-	gorm.io/gorm v1.24.0
+	gorm.io/driver/mysql v1.3.6
+	gorm.io/driver/postgres v1.4.5
+	gorm.io/gorm v1.24.1-0.20221019064659-5dd2bb482755
```

Comment:
gorm 1.24 has been released, we can upgrade gorm to a stable version first.
https://github.com/clusterpedia-io/clusterpedia/pull/441